CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [0.12.2] - TBD
+
+### Added
+- AWS KMS server Keymanager plugin (#2066)
+- Upstream Authority plugin that uses the Certificate Authority Service from Google Cloud Platform (#2172)
+- Improved error returned during attestation of agents (#2159)
+- AWS IID node attestor now supports running in a location with no public internet access available for the server (#2119)
+- The k8s notifier can now rotate Webhook CA Bundles (#2022)
+- Rate limiting on X509 signing and JWT signing can now be disabled (#2142)
+- Uptime metrics in server and agent (#2032)
+- Timeouts to calls to KeyManager plugins (#2044)
+- Logging when lookup of user by uid or group by gid fails in the `unix` WorkloadAttestor plugin (#2048)
+
+### Changed
```

Comment:
The items in this list don't read in the same way as the `Added` and `Fixed` sections, in other words, they don't form a complete, well formed sentence when prefixed with `Changed`.

Reply:
We should definitely go for consistency. That being said, I like the style in this Changed section better than the others :)

Reply:
Heh, I just looked at previous changelogs and sometimes we've added the prefix and others not, even within the same changelog section.

I also like the form that doesn't assume the prefix word. It reads better to me. I agree that whatever we do we should be consistent though, at a minimum within the same changelog entry :)